Add denormalized tables

ianalexmac opened the pull request:
This PR adds and organizes denormalized tables.

ianalexmac commented:
I'm noticing the column `capacity_mw`. Perhaps we'd like to change to `capacity` instead? We've dropped units from some other columns (such as those in `*_generation`), this is an opportunity to change those names before we write metadata etc.

ianalexmac commented:
Additional heads up: I changed table names around, aiming for brevity. I'm happy to have a discussion about this.

eldobbins commented:
- I *don't* have a problem with the table names
- I *do* have a problem with dropping units from the column names because there is the possibility of the metadata file being separated from the data file. When did they get dropped from generation? The copies I worked with before said `net_generation_mwh`
- I see that `communities` is not geojson in the public version, which I think is OK. This works for point datasets like this, but means I'm not sure about boroughs etc. I'll accept this now, but we might have to revisit depending on what data is going to be served via the explorer ultimately.
- `communities` is missing GNIS and PUMA code that are in the database but not shown in AEDG. That suggests another conversation about the scopes of the output files that might happen in the future - it is all data or only what we display?
- There are 4 datasets in the Jira card that are not here. If there are problems with those datasets, please document them here.
  - taxes
  - employment
  - census
  - rates

ianalexmac commented:
Perhaps I'm misremembering dropping units from column names in generation. I've looked around at output tables but can't find any examples. I could have sworn we had a discussion about this.

ianalexmac commented:
As for the missing tables compared to the Jira card, I forgot to include code in the dump script to add these tables, updated with commit [d87a2eb](https://github.com/acep-aedg/aedg-data-pond/pull/18/commits/d87a2ebff7e35dcf135c1e6d256fc4285e5b4efe)

ianalexmac commented:
> I see that communities is not geojson in the public version, which I think is OK. This works for point datasets like this, but means I'm not sure about boroughs etc. I'll accept this now, but we might have to revisit depending on what data is going to be served via the explorer ultimately.

We could just as easily output a GeoJSON with geometry for spatial tables such as `communities`, `boroughs` etc. I was imagining users would want to open these files in Excel, but maybe that's not the case.

ianalexmac commented:
> communities is missing GNIS and PUMA code that are in the database but not shown in AEDG. That suggests another conversation about the scopes of the output files that might happen in the future - it is all data or only what we display?

I dropped GNIS but could easily add it back alongside `fips_code`. It makes the table a little more busy but the clarity is probably more important. 

I dropped PUMA because I thought it was only a foreign key, but looking at `l2_pumas` reminds me that there are text strings we could use to normalize.

ianalexmac commented:
@eldobbins I added the fields `gnis_code` and `puma` to `public_communites` as well as changed `public_communities` from CSV to GeoJSON.

eldobbins commented:
@ianalexmac I do not think the data explorer can display GeoJSON files.  We may want to do that in the future, but not now. Please changes the communities file back to CSV.
